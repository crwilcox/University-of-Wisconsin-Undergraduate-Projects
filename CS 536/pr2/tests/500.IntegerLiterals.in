0
1
10
100
203
3458928
99999999999999999999
00
000
0123