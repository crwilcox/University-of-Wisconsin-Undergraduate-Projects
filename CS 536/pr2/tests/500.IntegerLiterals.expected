1:1:INT_LIT: 0
2:1:INT_LIT: 1
3:1:INT_LIT: 10
4:1:INT_LIT: 100
5:1:INT_LIT: 203
6:1:INT_LIT: 3458928
7:1:Integer out of range: 99999999999999999999
8:1:Integer has leading 0: 00
9:1:Integer has leading 0: 000
10:1:Integer has leading 0: 0123
